paper-checkbox.html

```diff
@@ -165,7 +165,7 @@
       this.checked = !this.checked;
 
       if (this.checked !== old) {
-        this.fire('iron-change');
+        this.fire('change');
```

Comment:
So I think the behavior of the old paper-radio-button is correct:

If a user clicks (or presses enter), fire both the 'change' and 'iron-change' event
If checked changes, just fire 'iron-change'

That's the behavior this cl achieves because checkedChange is still called in the _onMouseDown case. I think this is doing what you're suggesting.